tests/test_client/tests.py

```diff
@@ -202,6 +202,43 @@ def test_follow_relative_redirect_no_trailing_slash(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.request['PATH_INFO'], '/accounts/login/')
 
+    def test_follow_307_redirect(self):
```

Comment:
Could you combine this with `test_follow_308_redirect()` below and use `subTest()` to keep it DRY?